logger.gd

```diff
@@ -212,6 +212,7 @@ const ERROR = 4
 const STRATEGY_MUTE = 0
 const STRATEGY_PRINT = 1
 const STRATEGY_FILE = 2
+const STRATEGY_PRINT_AND_FILE = 3
```

Comment:
This should probably be:
```suggestion
const STRATEGY_PRINT_AND_FILE = STRATEGY_PRINT | STRATEGY_FILE
```
So that it stays valid if `STRATEGY_PRINT` or `STRATEGY_FILE` values change.